aspnetcore/security/enforcing-ssl.md

```diff
@@ -113,6 +129,13 @@ Per [OWASP](https://www.owasp.org/index.php/About_The_Open_Web_Application_Secur
 * The browser stores configuration for the domain that prevents sending any communication over HTTP. The browser forces all communication over HTTPS. 
 * The browser prevents the user from using untrusted or invalid certificates. The browser disables prompts that allow a user to temporarily trust such a certificate.
 
+HSTS is a client-side performance optimization to avoid an extra request to the server. HSTS limitations:
+
+* The client needing to support HSTS.
```

Comment:
This seems unclear ...

> The client needing to support HSTS.

... and prefer "requires" to "need"/"needs"/"needing"